### ISSUE TITLE
Dataset of source code

### ISSUE THREAD

**Issue by island255** [SoftSec-KAIST/BinKit#1]
Could you please release a dataset of all source code? Manually downloading all packages of the right version is time-consuming and error-prone.

By the way, this is an outstanding work for releasing such a complete binary code similarity detection dataset.

**Comment by 0xdkay** [comment]
Thanks for your interest.

BinKit downloads the source code of GNU packages and compiles them for each compile option.
You can check [gnu_compile_script.py](https://github.com/SoftSec-KAIST/BinKit/blob/master/gnu_compile_script.py) for downloading the code, and [compiled_package.sh](https://github.com/SoftSec-KAIST/BinKit/blob/master/compile_packages.sh) for its usage. For building, please check [README.md](https://github.com/SoftSec-KAIST/BinKit/blob/master/README.md#how-to-use).

BTW, compiling the dataset would take too much time, and this is why I share the pre-compiled dataset and tools.

**Comment by island255** [comment]
Thanks for your reply.

I am considering I maybe need to refer to source code to get a better understanding. Compiling such a big dataset is indeed a hard job.

**Comment by 0xdkay** [comment]
I'll also add more details on the README file to help :)

**Comment by island255** [comment]
Oh, thank you very much!

**Comment by 0xdkay** [comment]
Please check 8a3c04e. If you have any question, feel free to reopen this.

**Comment by island255** [comment]
I have successfully got the source code of the dataset. But it seems only to get the newest version of these packages. 

I am trying to execute helper/extract_functype.py in SoftSec-KAIST/TikNib. But I find the version of findutils in binary is 4.6.0, while the source I crawled is the newest, which is 4.7.0. 

I know specifying the exact version is time-consuming. So I only point this mismatch and remind the users.
